cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -517,9 +539,51 @@ impl ManifestPackageDescriptor {
     }
 }
 
+/// A map of service names to service definitions
+#[derive(
+    Debug,
+    Clone,
+    Serialize,
+    Deserialize,
+    Default,
+    PartialEq,
+    derive_more::Deref,
+    derive_more::DerefMut,
+)]
+#[cfg_attr(test, derive(proptest_derive::Arbitrary))]
+pub struct ManifestServices(
+    #[cfg_attr(
+        test,
+        proptest(
+            strategy = "proptest::collection::btree_map(proptest_alphanum_string(10), any::<ManifestServiceDescriptor>(), 0..3)"
+        )
+    )]
+    BTreeMap<String, ManifestServiceDescriptor>,
+);
+
+/// The definition of a service in a manifest
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, Hash)]
+#[cfg_attr(test, derive(proptest_derive::Arbitrary))]
+#[serde(rename_all = "kebab-case")]
+#[serde(deny_unknown_fields)]
+pub struct ManifestServiceDescriptor {
+    /// The command to run to start the service
+    pub command: String,
+    /// Service-specific environment variables
+    pub vars: Option<ManifestVariables>,
+}
+
 #[derive(Debug, Clone, Serialize, Deserialize, Default, PartialEq, Eq, Hash)]
 #[cfg_attr(test, derive(proptest_derive::Arbitrary))]
-pub struct ManifestVariables(BTreeMap<String, String>);
+pub struct ManifestVariables(
+    #[cfg_attr(
+        test,
+        proptest(
+            strategy = "proptest::collection::btree_map(proptest_alphanum_string(10), any::<String>(), 0..3)"
```

Comment:
nit: is it worth it to define that as a method?

Reply:
Done

Reply:
Done